Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessMetadata.java

```diff
@@ -89,4 +123,31 @@ public void preserve() {
             logger.error(e.getLocalizedMessage());
         }
     }
+
+    /**
+     * Check and return whether any further metadata can be added to the currently selected structure element or not.
+     *
+     * @return whether any further metadata can be added to currently selected structure element.
+     */
+    public boolean metadataAddableToStructureElement() throws InvalidMetadataValueException {
+        createProcessForm.getAddMetadataDialog().prepareAddableMetadataForStructure();
+        return !(createProcessForm.getAddMetadataDialog().getAddableMetadata().isEmpty());
+    }
+
+    /**
+     * Adds an empty table line with the given type.
```

Comment:
What "given type"? This function does not appear to have parameters. Please update JavaDoc.